packages/former/src/components/FormContent.vue

```diff
@@ -14,13 +14,29 @@
       </button>
     </div>
   </div>
+  <FormFieldSelector v-if="isFormFieldSelectorOpen">
+    <div ref="formFieldSelector" class="flex flex-col">
+      <button
+        v-for="type in availableFieldTypes"
+        :key="type"
+        :label="`select ${type} as type for the new input field`"
+        class="relative m-2 flex flex-col items-center rounded-xl border border-gray-400 bg-gray-200 p-1 hover:bg-gray-300"
+        @click="confirm(type)"
+      >
+        <span class="text-2xl">{{ type }}</span>
+      </button>
+    </div>
```

Comment:
@ShahwarZia I would move all this inside the `FormFieldSelector` component as well as `useConfirmDialog` logic. When a type has been selected we emit an event in `FormFieldSelector` about the selected type. 

This would make the redundant code obsolete between `FormContent.vue` and `FormKitEdit.vue`.

Reply:
@mariusheine I have moved this content inside the FormFieldSelector component